packages/appstore/src/stores/root-store.ts

```diff
@@ -6,20 +6,23 @@ export type CoreStoreTypes = {
     client: Record<string, any>;
```

Comment:
If I'm not mistaken, if we don’t know exact type of a value, we should use 'unknown' instead of 'any'.
we should avoid using 'any' at all.
correct me please

Reply:
we can not call any method on `unknown ` type, so we should use `any` to be able to call methods like is_suspended_mt5_demo_server and so on